ebpf_ffi/ebpf.cc

```diff
@@ -212,6 +212,19 @@ struct bpf_result ffi_get_map_elements(int map_fd, uint64_t map_size) {
   return serialize_proto(res);
 }
 
+struct bpf_result ffi_get_map_elements_fd_array(uint64_t fd_array_addr,
+                                                uint32_t idx,
+                                                uint64_t map_size) {
+  MapElements res;
+  std::vector<uint64_t> elements;
+  std::string error_message;
+
+  int *fd_array = reinterpret_cast<int *>(fd_array_addr);
```

Comment:
Though I see that the rest of the values are set to use int, if I understand correctly from the definition of the map, this should be an array of uint8_t? Would it make sense to start using the explicit type and slowly replace int with uint8_t?

Ignore if I am misunderstood though.

Reply:
the `fd_array` is an array of file descriptors which naturally have a type `int`. So this has to stay this way :)